build.gradle

```diff
@@ -55,14 +85,56 @@ subprojects {
     }
     clean { delete 'classpath' }
 
+    if (![project(":embulk-util-aws-credentials")].contains(project)) {
```

Comment:
Thank you. I fixed wrong indent and refactor around. It's minor thing but, this `if` condition is nice, because embulk-util-aws-credentials project needs `bintrayUpload` task but not need `gem` task. On the other hand, others need `gem` task but not need `bintrayUpload`.